metricbeat/mb/event.go

```diff
@@ -60,6 +61,12 @@ func (e *Event) BeatEvent(module, metricSet string, modifiers ...EventModifier)
 		e.MetricSetFields = nil
 	}
 
+	// Set index prefix
+	// TODO: does this break if user sets an index himself?
```

Comment:
@urso What will happen here with user defined index patterns? Does this get overwritten?